components/MeetingsList.js

```diff
@@ -1,7 +1,15 @@
-export default function Meetings({ meetings }) {
+import styledComponents from 'styled-components';
+
+const CardList = styledComponents.ul`
+display: grid;
+grid-gap: 20px;
+
+`;
+
+export default function MeetingsList({ meetingsList }) {
   return (
-    <ul>
-      {meetings.map(meeting => {
+    <CardList style={{ margin: '0', padding: '0' }}>
```

Comment:
if you use styled components, you should do your styling completely there and avoid doing single inline stylings additionally :)

Reply:
thx, changed it.